src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -85,6 +85,7 @@ public static string CreateEncodedSignature(string input, SigningCredentials sig
         /// <exception cref="ArgumentNullException"><paramref name="input"/> or <paramref name="signingCredentials"/> is null.</exception>
         public static string CreateEncodedSignature(string input, SigningCredentials signingCredentials, bool cacheProvider)
         {
+            // TODO create overload that takes a Span<byte> for the input
```

Comment:
>TODO create overload that takes a Span<byte> for the input [](http://example.com/codeflow?start=15&length=58)

do we need a github issue to track this?

Reply:
I have made the change in brentsch/JsonWebTokenHandler, which builds upon this.
I will remove the TODO.